bp-esa-frontend/src/components/dashboard.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import FilterSearchForm from './filterSearchForm';
+import ReportsTables from './reportsTables';
+
+class Dashboard extends Component {
+  render() {
+    const filterCriteria = [
+        { name: 'date', display: 'Date/month (is a range)' },
+        { name: 'partner', display: 'Partner'},
```

Comment:
When selected, Partner should pop up a search bar that allows partners to be searched for.